web-console/src/views/home-view/home-view.tsx

```diff
@@ -366,136 +393,161 @@ GROUP BY 1`,
+            title: 'Tasks',
+            loading: state.taskCountLoading,
+            content: (
+              <>
+                {Boolean(state.runningTaskCount) && (
+                  <p>{pluralIfNeeded(state.runningTaskCount, 'running task')}</p>
+                )}
+                {Boolean(state.pendingTaskCount) && (
+                  <p>{pluralIfNeeded(state.pendingTaskCount, 'pending task')}</p>
+                )}
+                {Boolean(state.successTaskCount) && (
+                  <p>{pluralIfNeeded(state.successTaskCount, 'successful task')}</p>
+                )}
+                {Boolean(state.waitingTaskCount) && (
+                  <p>{pluralIfNeeded(state.waitingTaskCount, 'waiting task')}</p>
+                )}
+                {Boolean(state.failedTaskCount) && (
+                  <p>{pluralIfNeeded(state.failedTaskCount, 'failed task')}</p>
+                )}
+                {!(
+                  Boolean(state.runningTaskCount) ||
+                  Boolean(state.pendingTaskCount) ||
+                  Boolean(state.successTaskCount) ||
+                  Boolean(state.waitingTaskCount) ||
+                  Boolean(state.failedTaskCount)
+                ) && <p>There are no tasks</p>}
+              </>
+            ),
+            error: state.taskCountError,
+          })}
+          {this.renderCard({
+            href: '#servers',
+            icon: IconNames.DATABASE,
+            title: 'Servers',
+            loading: state.serverCountLoading,
+            content: (
+              <>
+                <p>{`${pluralIfNeeded(state.overlordCount, 'overlord')}, ${pluralIfNeeded(
+                  state.coordinatorCount,
+                  'coordinator',
+                )}`}</p>
+                <p>{`${pluralIfNeeded(state.routerCount, 'router')}, ${pluralIfNeeded(
+                  state.brokerCount,
+                  'broker',
+                )}`}</p>
+                <p>{`${pluralIfNeeded(state.historicalCount, 'historical')}, ${pluralIfNeeded(
+                  state.middleManagerCount,
+                  'middle manager',
+                )}`}</p>
+                {Boolean(state.peonCount) && <p>{pluralIfNeeded(state.peonCount, 'peon')}</p>}
+              </>
+            ),
+            error: state.serverCountError,
+          })}
+          {this.renderCard({
+            href: '#lookups',
+            icon: IconNames.PROPERTIES,
+            title: 'Lookups',
+            loading: state.lookupsCountLoading,
+            content: (
+              <>
+                <p>
+                  {!state.lookupsUninitialized
+                    ? pluralIfNeeded(state.lookupsCount, 'lookup')
+                    : 'Lookups uninitialized'}
+                </p>
+              </>
+            ),
+            error: !state.lookupsUninitialized ? state.lookupsCountError : null,
+          })}
+        </div>
+        {!state.statusLoading && this.renderStatusDialog()}
```

Comment:
I think it is totally cool to put `{!state.statusLoading && this.renderStatusDialog()}` in the `div` no need for a wrapper fragment